documentation/UsingSDKWithWristUnits.md

```diff
@@ -0,0 +1,12 @@
+[Back to main page](../README.md)
+
+# Using SDK with Polar wrist units
+
+Polar wrist units such as Polar Ignite 3, Polar Pacer (Pro), Polar Grit X Pro 2 or Polar Vantage V3 support online streaming of some data types,
+altough some special user action is required to be able to start the streaming feature and get the data.
+
+> [!IMPORTANT]
+>
+> Although the namings might be confusing, enabling the SDK on the wrist unit is not starting [SDK Mode](./SdkModeExplained.md) with `startSdkMode()` function. SDK Mode is currently not supported in Polar watches.
```

Comment:
Please add a sentence or paragraph to explain what wrist unit SDK mode is to further clarify the difference from SDK Mode explained in ./SdkModeExplained.md